packages/altive_lints/lint_test/lints/avoid_single_child.dart

```diff
@@ -36,15 +36,21 @@ class MyWidget extends StatelessWidget {
           ],
         ),
         // expect_lint: avoid_single_child
+        const Stack(
+          children: [
+            Text('Hello'),
+          ],
+        ),
+        // expect_lint: avoid_single_child
         ListView(
           children: const [
-            Text('Hello'),
+            Text('World'),
           ],
         ),
         // expect_lint: avoid_single_child
         SliverList.list(
           children: const [
-            Text('World'),
+            Text('Hello'),
```

Comment:
![badge](https://img.shields.io/badge/review-nits-inactive)
Why did you swap these two strings?
Both `Hello` looks good.

Reply:
I made this change because I believe the text should be alternated, like "Hello" and "World," similar to how you would arrange them in a Column or Row. Specifically, I placed the Stack widget under the Wrap widget, so I wrote "Hello," and since the ListView widget is under the Stack, I decided "World" should be written to alternate the text.

The following commit adds a fix to make both Hello.
https://github.com/altive/altive_lints/pull/57/commits/a21c8a61cc3ef050f62eac18bf2cafad266a5dac

Reply:
I see, you thought about what would happen if it appeared on the screen!
Please comment why next time.